ironfish-rust/src/lib.rs

```diff
@@ -25,6 +25,7 @@ pub mod xchacha20poly1305;
 pub use {
     ironfish_frost::frost,
     ironfish_frost::participant,
+    ironfish_zkp::hex::{bytes_to_hex, hex_to_bytes, hex_to_vec_bytes},
```

Comment:
Instead of putting this line here, you could put it in `ironfish-rust/src/serializing/mod.rs`, so that you don't need to change all the imports everywhere.